sequoia/settings/offline_rl/setting.py

```diff
@@ -0,0 +1,101 @@
+import d3rlpy
+import gym
+from gym.wrappers import RecordEpisodeStatistics
+from matplotlib import pyplot as plt
+from sklearn.model_selection import train_test_split
+from typing import ClassVar, List, Tuple, Dict, Any
+
+from sequoia import Results
+from sequoia.settings.base import Setting
+from torch.utils.data import DataLoader
+from sequoia.settings.base import Method
+from dataclasses import dataclass
+
+from simple_parsing.helpers import choice
+
+
+
+@dataclass
+class OfflineRLResults(Results):
+
+    # TODO: Write these methods
+    def summary(self) -> str:
+        pass
```

Comment:
Should return a string description of these results. Worst case, just return `str(asdict(self))` or something.